### DISCUSSION TITLE
Very annoying HA blocks

### DISCUSSION THREAD

**Discussion by cafra74** [mampfes/hacs_waste_collection_schedule#2868] [Q&A]
Sorry, I wanted to ask those who are certainly more expert than me, what could be the cause of these continuous errors that block my HA?
It's on a virtual machine (VirtualBox 7.1.0) and has Core 2024.10.2
I know it would not be appropriate to run HA under virtualization but this is my first approach and given the problems it will certainly not be my final solution
If I restart the virtual machine it works for a short while but then I can't even reach it locally anymore.
Sorry for my english and thank you all for your help

![immagine_2024-10-18_141221042](https://github.com/user-attachments/assets/7b9f1ec8-2139-4b77-b57b-89cf01eb9232)

**Comment by 5ila5** [comment]
I don't think I can help you with this. I doubt it has anything to do with this integration.
You probably have better luck posting this issue here: https://community.home-assistant.io/
